src/cascadia/TerminalSettingsModel/ActionMap.h

```diff
@@ -44,14 +44,12 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
     {
         bool operator()(const Control::KeyChord& lhs, const Control::KeyChord& rhs) const
         {
-            return lhs.Modifiers() == rhs.Modifiers() && lhs.Vkey() == rhs.Vkey() && lhs.ScanCode() == rhs.ScanCode();
+            return lhs.Modifiers() == rhs.Modifiers() && lhs.Vkey() == rhs.Vkey();

```

Comment:
This seems like an actual change (not just a part of the conversion to STL). Are you sure we want this?

Reply:
Ah woops that one sneaked in - that's part of an actual PR.